### ISSUE TITLE
os.walk() example of an implementation of shutil.rmtree() is incomplete

### ISSUE THREAD

**Issue by vwheeler63** [python/cpython#126055]
# Documentation

In this section:

https://docs.python.org/3.9/library/os.html?highlight=os%20walk

there is a code example at the bottom that claims to be "a simple implementation of `shutil.rmtree()`".  However it is incomplete.  Reason:  `shutil.rmtree()` also removes the passed directory whereas the code shown will not.  To make it complete, add this line at the end of it *after* the top (highest) `for` loop:
```Python
os.rmdir(top)
```

That will make the whole code example look like this:
```Python
# Delete everything reachable from the directory named in "top",
# assuming there are no symbolic links.
# CAUTION:  This is dangerous!  For example, if top == '/', it
# could delete all your disk files.
import os
for root, dirs, files in os.walk(top, topdown=False):
    for name in files:
        os.remove(os.path.join(root, name))
    for name in dirs:
        os.rmdir(os.path.join(root, name))

os.rmdir(top)
```

### Linked PRs
* gh-126067
* gh-126199
* gh-126200

**Comment by vwheeler63** [comment]
> PR is welcomed!

Wow!  I am honored!  Thank you for the invite!  I didn't know it was open for contribution!  Fix submitted in PR #126067.

**Comment by picnixz** [comment]
Anyone is free to contribute whether it is by opening issues or addressing them in a PR (or reviewing them)! You should just read the https://devguide.python.org/ and the CONTRIBUTING.rst file in the repository if you want to submit a PR in general!
